src/styles/blocks/page.scss

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
add also border-box

```
html {
  box-sizing: border-box;
}

*,
*::after,
*::before {
  box-sizing: inherit;
}

body {
  margin: 0;
}
```

Reply:
Add also border-box.

Reply:
Deleted border-box becouse test is failed.